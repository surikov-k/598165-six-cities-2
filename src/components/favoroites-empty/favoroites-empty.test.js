import React from 'react';
import renderer from 'react-test-renderer';
import FavoritesEmpty from './favoroites-empty';

jest.mock(`react-router-dom`,
    () => ({
      Link: () => null,
    }));

describe(`<FavoritesEmpty/>`, () => {
  it(`renders correctly`, () => {
    const component = renderer
      .create(<FavoritesEmpty
        user={{
          id: 0,
          email: ``
        }}
        isAuthorizationRequired={false}
      />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});

