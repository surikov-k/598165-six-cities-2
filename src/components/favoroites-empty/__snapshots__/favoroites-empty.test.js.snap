// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoritesEmpty/> renders correctly 1`] = `
<div
  className="page page--favorites-empty"
>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        />
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            />
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--favorites page__main--favorites-empty"
  >
    <div
      className="page__favorites-container container"
    >
      <section
        className="favorites favorites--empty"
      >
        <h1
          className="visually-hidden"
        >
          Favorites (empty)
        </h1>
        <div
          className="favorites__status-wrapper"
        >
          <b
            className="favorites__status"
          >
            Nothing yet saved.
          </b>
          <p
            className="favorites__status-description"
          >
            Save properties to narrow down search or plan yor future trips.
          </p>
        </div>
      </section>
    </div>
  </main>
  <footer
    className="footer"
  />
</div>
`;
